tests/src/test/scala/com/nvidia/spark/rapids/BroadcastHashJoinSuite.scala

```diff
@@ -20,7 +20,7 @@ import com.nvidia.spark.rapids.TestUtils.findOperator
 
 import org.apache.spark.SparkConf
 import org.apache.spark.sql.functions.broadcast
-import org.apache.spark.sql.rapids.execution.GpuHashJoin
+import org.apache.spark.sql.rapids.execution.{GpuBroadcastHashJoinExec, GpuHashJoin}
```

Comment:
import changes only?

Reply:
I moved GpuBroadcastHashJoinExec to a different package when I shimmed it, so that explains this. I'm a little uncertain of what the right package should be for it?

Reply:
Oh sorry, I missed the package relocation earlier.  It doesn't look like it needed to be relocated based on the changes to that class?

Reply:
Ah yes, some shims need it.  That's fine, we can leave it in the Spark package.